kpc/tests/__init__.py

```diff
@@ -0,0 +1,14 @@
+from model_mommy import mommy
+
+
+# setup model_mommy for django-localflavor
+def _gen_zipcode():
+    return "00000-0000"
+
+
+def _gen_state():
+    return 'NY'
+
+
+mommy.generators.add('localflavor.us.models.USZipCodeField', _gen_zipcode)
+mommy.generators.add('localflavor.us.models.USStateField', _gen_state)
```

Comment:
What do you think about using lambda expressions here? I'm generally in favor of named functions, but it looks like we _don't_ want these to be accessed outside of these lines.
```python
mommy.generators.add('localflavor.us.models.USZipCodeField', lambda: "00000-0000")
mommy.generators.add('localflavor.us.models.USStateField', lambda: "NY")
```